.github/workflows/application.yml

```diff
@@ -24,22 +24,15 @@ jobs:
     static-checks:
         runs-on: ubuntu-latest
 
-        name: "Static checks (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}), API Platform ${{ matrix.api-platform }})"
+        name: "Static checks (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }})"
 
         timeout-minutes: 10
 
         strategy:
             fail-fast: false
             matrix:
                 php: ["8.0", "8.1"]
-                symfony: ["^5.4"]
-                api-platform: ["^2.7"]
-
-                include:
-                    -
-                        php: "8.1"
-                        symfony: "^5.4"
-                        api-platform: "~2.6.0"
```

Comment:
why removing this api-platform checks?

Reply:
When changing the ApiPlatform version there were problems with the validation of package versions (we would also need to change it inside the ApiBundle which seemed weird). I agree we should take care of this problem 👍 but maybe we could do it after the merge to not make iteration on this one even longer 😄